### PR TITLE
[infra] Print exact reproduction commands in build failures (#3104).

### PR DESCRIPTION
**Pull request by Dor1s**
This also includes some formatting changes. Sorry I didn't realize that these scripts were not formatted. I can separate formatting and the actual change, but that would have to be done manually, as I don't have a non-formatted commit even locally.

### CODE REVIEW

**Comment by Dor1s** [comment]
I'll run this on GCB right now and post log examples.

**Comment by Dor1s** [comment]
Travis reports lint failure because it doesn't install dependencies for GCB scripts, plus some other naming / commenting concerns.

**Comment by Dor1s** [comment]
Let me remove `&&` from the commands as they aren't fully copy-pastable anyway, below are some examples:

- simple build failure in default config: https://console.cloud.google.com/logs/viewer?resource=build%2Fbuild_id%2Ffdd96f43-4e3b-4ffe-b808-34e6045ad128&project=oss-fuzz&minLogLevel=0&expandAll=false&timestamp=2020-01-15T18:53:55.598000000Z&customFacets=&limitCustomFacetWidth=true&dateRangeStart=2020-01-15T17:53:55.850Z&dateRangeEnd=2020-01-15T18:53:55.850Z&interval=PT1H&scrollTimestamp=2020-01-15T18:22:34.336157550Z
- simple build failure in non-default sanitizer/engine config: https://console.cloud.google.com/logs/viewer?resource=build%2Fbuild_id%2F5462827a-3211-48f0-84fb-1dc2bfe0188f&project=oss-fuzz&minLogLevel=0&expandAll=false&timestamp=2020-01-15T18:55:13.507000000Z&customFacets=&limitCustomFacetWidth=true&dateRangeStart=2020-01-15T17:55:13.758Z&dateRangeEnd=2020-01-15T18:55:13.758Z&interval=PT1H&scrollTimestamp=2020-01-15T18:29:39.162784015Z
- bad build check failure: https://console.cloud.google.com/logs/viewer?resource=build%2Fbuild_id%2Fe28b5c38-5a28-4181-b393-80648af50c50&project=oss-fuzz&minLogLevel=0&expandAll=false&timestamp=2020-01-15T18:55:54.544000000Z&customFacets=&limitCustomFacetWidth=true&dateRangeStart=2020-01-15T17:55:54.797Z&dateRangeEnd=2020-01-15T18:55:54.797Z&interval=PT1H&scrollTimestamp=2020-01-15T18:23:59.203881902Z
- coverage build failure: https://console.cloud.google.com/logs/viewer?resource=build%2Fbuild_id%2F1c89b35a-5319-41c6-99f4-2eb673160470&project=oss-fuzz&minLogLevel=0&expandAll=false&timestamp=2020-01-15T18:56:54.897000000Z&customFacets=&limitCustomFacetWidth=true&dateRangeStart=2020-01-15T17:56:55.148Z&dateRangeEnd=2020-01-15T18:56:55.148Z&interval=PT1H&scrollTimestamp=2020-01-15T18:27:13.106232083Z
- coverage generation failure (due to missing corpus backup in this particular case): https://console.cloud.google.com/logs/viewer?resource=build%2Fbuild_id%2F88b6f498-4a95-4e3a-bdfd-1beb910a0107&project=oss-fuzz&minLogLevel=0&expandAll=false&timestamp=2020-01-15T18:56:22.901000000Z&customFacets=&limitCustomFacetWidth=true&dateRangeStart=2020-01-15T17:56:23.153Z&dateRangeEnd=2020-01-15T18:56:23.153Z&interval=PT1H&scrollTimestamp=2020-01-15T18:27:12.497706644Z

**Comment by Dor1s** [comment]
ready for review @inferno-chromium

File: infra/gcb/build_project.py [hunk 1]
```diff
@@ -233,8 +233,13 @@ def get_build_steps(project_dir):
         if not workdir:
           workdir = '/src'
 
-        failure_msg = ('FAILED IN THE FOLLOWING CONFIG: --sanitizer {0} '
-                       '--engine {1}').format(sanitizer, fuzzing_engine)
+        failure_msg = ('*' * 80 + '\nFailed to build.\nTo reproduce, run:\n'
+                       'python infra/helper.py build_project {0}\n'
```
Comment (inferno-chromium) [inline infra/gcb/build_project.py]: nit:lets use named args, for anything > 1 argment

Comment (Dor1s) [reply infra/gcb/build_project.py]: Done!

**Pull request merged**